src/app/viewing/messages/messages.component.css

```diff
@@ -0,0 +1,47 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+
+.toggle-switch {
+    color: #999999;
```

Comment:
Use a main.css variable colour or add a colour to main.css so that our colour scheme can be consistent